src/gax/src/paginator.rs

```diff
@@ -397,4 +403,89 @@ mod tests {
         assert_eq!(sdk_util::extract_token(&Some("abc".to_string())), "abc");
         assert_eq!(sdk_util::extract_token(&None::<String>), "");
     }
+
+    #[cfg(feature = "unstable-stream")]
+    #[tokio::test]
+    async fn test_paginator_to_stream() {
+        let responses = vec![
+            TestResponse {
+                items: vec![
+                    PageItem {
+                        name: "item1".to_string(),
+                    },
+                    PageItem {
+                        name: "item2".to_string(),
+                    },
+                ],
+                next_page_token: "token1".to_string(),
+            },
+            TestResponse {
+                items: vec![PageItem {
+                    name: "item3".to_string(),
+                }],
+                next_page_token: "".to_string(),
+            },
+        ];
+
+        let client = Client {
+            inner: Arc::new(InnerClient {
+                data: Arc::new(Mutex::new(responses)),
+            }),
+        };
+        let mut resps = vec![];
+        let mut stream = client.list_rpc_stream(TestRequest::default()).to_stream();
+        while let Some(resp) = stream.next().await {
+            if let Ok(resp) = resp {
+                resps.push(resp)
+            }
+        }
+        assert_eq!(resps.len(), 2);
+        assert_eq!(resps[0].items[0].name, "item1");
+        assert_eq!(resps[0].items[1].name, "item2");
+        assert_eq!(resps[1].items[0].name, "item3");
+    }
+
+    #[cfg(feature = "unstable-stream")]
+    #[tokio::test]
+    async fn test_item_paginator_to_stream() {
+        let responses = vec![
+            TestResponse {
+                items: vec![
+                    PageItem {
+                        name: "item1".to_string(),
+                    },
+                    PageItem {
+                        name: "item2".to_string(),
+                    },
+                ],
+                next_page_token: "token1".to_string(),
+            },
+            TestResponse {
+                items: vec![PageItem {
+                    name: "item3".to_string(),
+                }],
+                next_page_token: "".to_string(),
+            },
+        ];
+
+        let client = Client {
+            inner: Arc::new(InnerClient {
+                data: Arc::new(Mutex::new(responses)),
+            }),
+        };
+        let mut items = vec![];
+        let mut stream = client
+            .list_rpc_stream(TestRequest::default())
+            .items()
+            .to_stream();
```

Comment:
codecov.io seems confused about the coverage for `to_stream()`.  is this code running?

Reply:
Hm, it should be exercised here:

https://github.com/googleapis/google-cloud-rust/blob/66824576babf09a168207a74695fdd1758bec70a/.github/workflows/sdk.yaml#L47

Locally:

```sh
$ cargo test --workspace | grep to_stream
test paginator::tests::test_item_paginator_to_stream ... ok
test paginator::tests::test_paginator_to_stream ... ok
```

Reply:
FWIW, I believe the  coverage build uses a different set of options:

https://github.com/googleapis/google-cloud-rust/blob/66824576babf09a168207a74695fdd1758bec70a/.github/workflows/sdk.yaml#L69-L74

Reply:
oh, d'uh. I misread the yaml.